config/config_test.go

```diff
@@ -25,9 +25,8 @@ import (
 var configStr = `
 {
     "handler": {
-        "slack": {
-            "channel": "slack_channel",
-            "token": "slack_token"
+        "webhook": {
+            "url": "http://localhost:8080"
         }
```

Comment:
handler.slack testing seems to have been removed, it would be nice to keep it alongside the added one

Reply:
I just wanted to use the easier method to debug, which I thought it was Webhook.  Adding slack back.